pose_constraints_pipeline:
  ros__parameters:
    group_name: ur_robot
    joint_names:
    - ur10e_shoulder_pan_joint
    - ur10e_shoulder_lift_joint
    - ur10e_elbow_joint
    - ur10e_wrist_1_joint
    - ur10e_wrist_2_joint
    - ur10e_wrist_3_joint

    wait_joint_states_sec: 5.0
    max_joint_state_age_sec: 1.0
    joint_states_topic: /joint_states

    allowed_planning_time: 60.0
    num_planning_attempts: 3
    joint_tolerance: 0.001
    max_velocity_scaling_factor: 0.1
    max_acceleration_scaling_factor: 0.1

    plan_action_name: /plan_with_constraints
    time_param_action_name: /apply_time_parametrization
    execute_action_name: /execute_trajectory

    verbose: false
    stop_on_error: true
    dry_run: false

